lib/utils/validate-addons.js

```diff
@@ -1,21 +1,18 @@
 const exists = require('./npm-exists');
-const parse = require('../parser/index');
+const resolvePackages = require('./resolve-packages');
 
 module.exports = function validateAddons(addon) {
-	Error.stackTraceLimit = 3;
-	return addon.filter( pkg => {
-		if(pkg.slice(0,13) !== 'webpack-addon') {
-			throw new TypeError('Package isn\'t a valid name\n' +
-			'It should be prefixed with \'webpack-addon\'');
-		} else if(pkg.length == 13 && pkg.slice(0,13) === 'webpack-addon') {
-			throw new TypeError('\'webpack-addon\' is not a valid addon');
-		}
+	Error.stackTraceLimit = 4;
+	return addon.map( pkg => {
+		//eslint-disable-next-line
 		exists(pkg).then( (moduleExists) => {
-			!moduleExists ? (Error.stackTraceLimit) = 0 : (Error.stackTraceLimit = 30);
+			!moduleExists ? (Error.stackTraceLimit = 0) : (Error.stackTraceLimit = 30);
```

Comment:
Can the `stackTraceLimit` be a variable with default value of 4 and in the end you'll assign it to `Error.stackTraceLimit`?